### ISSUE TITLE
Warning: Overlapping range

### ISSUE THREAD

**Issue by daschuer** [mixxxdj/mixxx#11545]
### Bug Description

I have see this warning when shutting down Mixxx with an m4a file playing: 
```
warning [CachingReaderWorker 1] ReadAheadFrameBuffer - Overlapping range [960 -> 1024) in output buffer [1024 -> 8192) with input buffer [960 -> 1984)
```

### Version

2.4-alpha

### OS

Ubunti 20.4 Focal

**Comment by daschuer** [comment]
This might be related to a bugfix for ffmpeg 4.4 introduced in 2.3.1 here: 
https://github.com/mixxxdj/mixxx/issues/10464
Ubuntu Focal uses ffmpeg 2.2

**Comment by daschuer** [comment]
I have seen also warnings about a shifted sound start cue along with this issue.

**Comment by ywwg** [comment]
what is the effect of the warning? why is this a blocker for 2.3.6?

**Comment by daschuer** [comment]
The effect is that a single chunk is shifted. The overlapping samples are dropped and the gap is filled with silence. 

I don't consider it a strict blocker because only older distros are affected, but it is part of the list of regressions, (Introduced in 2.3.1) and we have already a PR for it.

**Comment by Swiftb0y** [comment]
closed by #11760
